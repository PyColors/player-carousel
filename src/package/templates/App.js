import React from "react";
import Header from "../components/organisms/Header";
import Nav from "../components/organisms/Nav";
import Footer from "../components/organisms/Footer";

// muicss
import Container from "muicss/lib/react/container";

import './App.scss';

const App = () => (
    <main>
        <Container>
            <Header/>
            <Nav/>
            <Footer/>
        </Container>
    </main>
);
export default App;
