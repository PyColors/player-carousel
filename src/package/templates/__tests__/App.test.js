import React from "react";
import App from "../App";

it("Should renders App page without exploding", () => {
    const wrapper = shallow(<App>Hello Jest!</App>);
    expect(wrapper).toMatchSnapshot();
});
