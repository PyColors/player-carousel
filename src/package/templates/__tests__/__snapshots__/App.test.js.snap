// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders App page without exploding 1`] = `
<main>
  <Container
    className=""
    fluid={false}
  >
    <Header />
    <Nav />
    <Footer />
  </Container>
</main>
`;
